frontend/components/base/base-tabs/BaseTabs.vue

```diff
@@ -0,0 +1,107 @@
+<!--
+  - coding=utf-8
+  - Copyright 2021-present, the Recognai S.L. team.
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -     http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -->
+
+<template>
+  <ul class="tabs">
+    <li class="tab">
+      <button
+        v-for="tab in tabs"
+        :key="tab.id"
+        :class="['tab__button', getTabClass(tab)]"
+        @click="changeTab(tab.id)"
+      >
+        <span>{{ tab.name }}</span>
+      </button>
+    </li>
```

Comment:
1/ destructuration of tab, id and name from tab
2/ pass tab as v-text into <span />